action.yml

```diff
@@ -1,16 +1,39 @@
-name: 'JavaScript Action Template'
-
-description: 'Add description here'
+name: Update Deployment Board
+description: Adds deployment Issues to an Automated Deployment Project Board for visual tracking of what has been deployed to each environment.
 
 inputs:
-  input-1:
-    description: 'Add a description here'
+  workflow-actor:
+    description: The GitHub user who triggered the workflow
+    required: true
+  token:
+    description: 'A token with `repo_deployment` permissions to create and update issues, workflows using this action should be granted `permissions` of `deployments: write`'
+    required: true
+  environment:
+    description: The environment the release was deployed to, i.e. [DEV|QA|STAGE|DEMO|UAT|PROD]
```

Comment:
```suggestion
    description: The name of a GitHub environment the release was deployed to, i.e. [Dev|QA|Stage|Demo|UAT|Prod]. It will be set in the standard `environment` property of the deployment object.
```